integration-cli/docker_cli_service_health_test.go

```diff
@@ -132,60 +132,3 @@ func (s *DockerSwarmSuite) TestServiceHealthStart(c *check.C) {
 		return task.Status.State, nil
 	}, checker.Equals, swarm.TaskStateRunning)
 }
-
-// start a service whose task is unhealthy at beginning
-// its tasks should be blocked in starting stage, until health check is passed
-func (s *DockerSwarmSuite) TestServiceHealthUpdate(c *check.C) {
```

Comment:
Ahh, I see. It's a duplicate.

I suspect it should be testing something else, but the line wasn't updated after the test was copied.